src/Components/v2/Carousel.tsx

```diff
@@ -167,7 +170,7 @@ const CarouselContainer = styled.div<{ height?: number }>`
 `
 
 export const ArrowButton = styled(Flex)<
-  LeftProps & RightProps & { height?: number }
+  LeftProps & RightProps & { height?: number; arrowHeight?: number }
```

Comment:
I pass this `arrowHeight` prop here to be able to be able to center the carousel arrow based on the image: 

![Screen Shot 2019-04-09 at 2 18 33 PM](https://user-images.githubusercontent.com/5201004/55824703-65db1780-5ad2-11e9-813d-b479bb6c1b64.png)

I don't think that's super ideal. Do you have any other ideas to achieve this? We are trying to apply the desired styles where we use the component [here](https://github.com/artsy/reaction/blob/6cb1ab7fee49807ea6c67fc0e81b8aef640b2a7d/src/Components/Artist/ArtistCollectionsRail/ArtistCollectionsRail.tsx#L75-L78) but those styles aren't getting passed down.

Reply:
I believe the earlier implementation for arrow height used styled components, this can probably be removed now if we are setting via the carousel.

Reply:
@eessex I didn't remove it yet because I wasn't set on this implementation. I'm not convinced this is the best approach.

Reply:
@xtina-starr - I've opened a PR here so that these sorts of modifications can be more general: https://github.com/artsy/reaction/pull/2283. If this looks good to you feel free to merge and then rebase.